gym_ai2thor/config_files/config_example.json

```diff
@@ -18,6 +18,6 @@
     "resolution": [128, 128],
     "task": {
         "task_name": "PickUp",
-        "target_object": "Mug"
+        "target_objects": {"Mug": 1}
```

Comment:
I find it more tiresome to make 2 lists: `["Apple", "Mug", "Pencil"], [2, 1, 2]` rather than put both of them together in a dictionary. Having a dictionary forces you to write down how much reward you want to give. Explicit better than implicit

Reply:
yeah but maybe we should allow just a list which implies all objects are 1 reward?

Reply:
We can discuss this but I don't see it as a good option

Reply:
For my natural language pick up or look at tasks, I'm currently not doing this but we'll see if it is necessary

Reply:
Ok please confirm that this is the way to go. Explicit rewards and no option for list of 1 rewards.

Reply:
open a thread in slack